charts/unified-chart/values.yaml

```diff
@@ -64,35 +78,86 @@ worker:
   enabled: false
   command: ["/bin/bash"]
   args: ["entrypoint.sh", "worker"]
-  env:
-    # - name: FRONTEGG_IS_SERVICE_OFFLINE_PROCESS
-    #   value: "true"
+  replicaCount: 1
+  autoscaling:
+    minReplicas: 1
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 50
+    targetMemoryUtilizationPercentage: 75
+    scaledObject:
+      enabled: false
+      triggers: []
+      advanced:
+        horizontalPodAutoscalerConfig:
+          name: '{{ include "unified.worker.name" $ }}'
+  env: []
+  livenessProbe:
+    failureThreshold: 3
+    httpGet:
+      path: /health
+      port: http
+    initialDelaySeconds: 40
+    periodSeconds: 60
+  readinessProbe:
+    failureThreshold: 1
+    httpGet:
+      path: /health
+      port: http
+    initialDelaySeconds: 5
+    periodSeconds: 5
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
 
-service:
-  type: ClusterIP
+highPriority:
+  enabled: false
+  labels:
+    priority: "high"
+  replicaCount: 1
+  env: []
+  serviceProfile:
```

Comment:
Service profile tend to be super long, If you will put it here it will make the file a bit unreadable

Reply:
does linkerd supports loading it from config map or something ?